app/uploaders/banner_image_uploader.rb

```diff
@@ -0,0 +1,8 @@
+class BannerImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
```

Comment:
To be honest, I only brought this in because Sufia's [`AvatarUploader`](https://github.com/projecthydra/sufia/blob/master/app/uploaders/sufia/avatar_uploader.rb
) does the same. I wasn't sure of it's importance, and forgot to ask.
 
Is there any situation where there could be multiple `Site` objects (and therefore multiple background images)?  I only ask because by default `CarrierWave` just stores all uploads under `public/uploads/`... I wasn't sure if there's any situation were we need these stored in different locations per `Site` (in which case we may need a custom storage path).